freckle-kafka/library/Freckle/App/Kafka/Producer.hs

```diff
@@ -136,30 +137,44 @@ produceKeyedOn
   -> NonEmpty value
   -> (value -> key)
   -> m ()
-produceKeyedOn prTopic values keyF = traced $ do
-  logDebugNS "kafka" $ "Producing Kafka events" :# ["events" .= values]
+produceKeyedOn prTopic values keyF = produce prTopic (NE.map mkProducerRecord values)
+ where
+  mkProducerRecord value =
+    ProducerRecord
+      { prTopic
+      , prPartition = UnassignedPartition
+      , prKey = Just $ BSL.toStrict $ encode $ keyF value
+      , prValue = Just $ BSL.toStrict $ encode value
+      , prHeaders = mempty
+      }
+
+-- | Produce/Emit a message to a Kafka topic, accepts a list of ProducerRecords (Kafka messages)
```

Comment:
Also do we need the `accepts a list of ProducerRecords (Kafka messages)`? It's in the signature after all

Reply:
Prolly not